src/gui/optionsdialog.cpp

```diff
@@ -575,6 +608,14 @@ void OptionsDialog::saveOptions()
     pref->setDontConfirmAutoExit(!m_ui->checkProgramAutoExitConfirm->isChecked());
     pref->setPreventFromSuspendWhenDownloading(m_ui->checkPreventFromSuspendWhenDownloading->isChecked());
     pref->setPreventFromSuspendWhenSeeding(m_ui->checkPreventFromSuspendWhenSeeding->isChecked());
+    if (!m_qssFilePath.isEmpty() &&
+        (m_ui->comboTheme->currentIndex() == 1)) {
+        // only install Theme if Current Selection is "Custom" so that
+        // previous "Custom" theme is not removed
+        UIThemeManager::instance()->installTheme(m_qssFilePath);
+        m_qssFilePath.clear();
```

Comment:
Redundant action in destructor, isn't it?

Reply:
idk, it's done in `saveoptions` called with every `accept`?

Reply:
well before some commit I was using empty `m_qssFilePath` as some sort of flag, now it's not necessary.

Reply:
Sorry, I misread it.